### DISCUSSION TITLE
[Grafana][Loki plugin] customize log query example in Kick start your query

### DISCUSSION THREAD

**Discussion by c609942867** [grafana/grafana#67483] [Plugins]
Hi Grafana team,
Could you provide add/remove button in kick start your query for admin to add customized query examples.
![image](https://user-images.githubusercontent.com/71490223/235115473-625cdc55-50ee-46e8-97b1-4cda457b2e50.png)

There are two benefits.
- beginners can understand how to use Loki with detailed user cases. The use cases can be designed based on business by admin.
- developers don’t need start one query from scratch

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82747. Feel free to continue the discussion around this there. Thank you!
